### ISSUE TITLE
User hitting 408 on 2.3-alpha7, meeting number on bbb-web higher than on mongo

### ISSUE THREAD

**Issue by antobinary** [bigbluebutton/bigbluebutton#11495]
**Describe the bug**
We are sporadically hitting a case where user authentication cannot proceed, resulting in
 * timeout and 408 error on the client
 * meeting not being handled on bbb-html5, so no meeting traces in Mongo
 * meeting kept on file in bbb-web, resulting in high count on getMeetings() response

**To Reproduce**
Not sure. Seems to happen when a meeting is created and first user joins right away

I have also seen it when a server has a number of ongoing sessions and parts of bbb-html5 are restarted, though not easily reproducible.

**Expected behavior**
Meeting should be created and joined fine. Meetings count in Mongo and in bbb-web should match

**Actual behavior**
408 error for user, number of meetings on Mongo lower than on bbb-web

**Screenshots**
![Screenshot from 2021-02-24 16-56-15](https://user-images.githubusercontent.com/6312397/109071066-47c83180-76c1-11eb-91ec-887cec51e8a1.png)

**BBB version (optional):**
2.3-alpha7

```
root@demo3:/var/log# grep lacyttqrzndtz3zb syslog.3 
Feb 19 17:43:12 demo3 run-prod.sh[15197]: 2021-02-19T17:43:12.051Z INFO  o.b.web.controllers.ApiController -  --analytics-- data={"extMeetingid":"966d66c049d23a165e6a8256775b5314bcef9a13","logCode":"join_api","name":"Support","sessionToken":"lacyttqrzndtz3zb","description":"Handle JOIN API.","meetingid":"1788b5073230bd8608a801d692817fa7a6b7d0de-1613756122885","userid":"w_dg2sv0roxghy"}
Feb 19 17:43:12 demo3 run-prod.sh[15197]: 2021-02-19T17:43:12.051Z INFO  o.b.web.controllers.ApiController - Redirecting to https://demo3.bigbluebutton.org/html5client/join?sessionToken=lacyttqrzndtz3zb
Feb 19 17:43:13 demo3 run-prod.sh[15197]: 2021-02-19T17:43:13.087Z INFO  o.b.web.controllers.ApiController - Token lacyttqrzndtz3zb is valid
Feb 19 17:43:13 demo3 run-prod.sh[15197]: 2021-02-19T17:43:13.088Z INFO  o.b.web.controllers.ApiController -  --analytics-- data={"extMeetingid":"966d66c049d23a165e6a8256775b5314bcef9a13","logCode":"handle_enter_api","name":"Support","sessionToken":"lacyttqrzndtz3zb","description":"Handling ENTER API.","meetingid":"1788b5073230bd8608a801d692817fa7a6b7d0de-1613756122885","userid":"w_dg2sv0roxghy"}
```

**Comment by antobinary** [comment]
Possibly related to #11650

**Comment by antobinary** [comment]
Fixed by #11650
